documentation/modules/proc-configuring-kafka-user.adoc

```diff
@@ -22,23 +22,24 @@ When you configure the `KafkaUser` authentication and authorization mechanisms,
 This procedure shows how a user is created with TLS authentication.
 You can also create a user with SCRAM-SHA authentication.
 
-The authentication required depends on the xref:assembly-kafka-broker-listener-authentication-deployment-configuration-kafka[type of authentication configured for the Kafka broker listener].
+The authentication required depends on the xref:con-securing-kafka-authentication-{context}[type of authentication configured for the Kafka broker listener].
 
 NOTE: Authentication between Kafka users and Kafka brokers depends on the authentication settings for each.
 For example, it is not possible to authenticate a user with TLS if it is not also enabled in the Kafka configuration.
 
 .Prerequisites
 
-* A running Kafka cluster xref:con-mutual-tls-authentication-deployment-configuration-kafka[configured with a Kafka broker listener using TLS authentication and encryption].
+* A running Kafka cluster xref:con-mutual-tls-authentication-{context}[configured with a Kafka broker listener using TLS authentication and encryption].
 * A running User Operator (typically xref:assembly-kafka-entity-operator-deployment-configuration-kafka[deployed with the Entity Operator]).
 
-If you are using SCRAM-SHA authentication, you need a running Kafka cluster xref:con-scram-sha-authentication-deployment-configuration-kafka[configured with a Kafka broker listener using SCRAM-SHA authentication].
+If you are using SCRAM-SHA authentication, you need a running Kafka cluster xref:con-scram-sha-authentication-{context}[configured with a Kafka broker listener using SCRAM-SHA authentication].
```

Comment:
This seems a bit out of place. I think we should replace it with something generic such as:

```suggestion
The authentication type in `KafkaUser` should match the authentication configured in `Kafka` brokers.
```